jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -223,6 +223,31 @@ def test_parse_jwst_nircam_level2(self, imviz_app):
             assert isinstance(data.coords, GWCS)
             assert comp.units == 'MJy/sr'
 
+        # Test duplicate label functionality
+        with fits.open(filename) as pf:
```

Comment:
I don't see why we cannot test with the existing `pf` above. Nice trick with the `.clear()`!
```suggestion
            # Test duplicate label functionality
```

Reply:
Fixed this. Uses a single `pf` now.